packages/valory/skills/abstract_round_abci/test_tools/base.py

```diff
@@ -132,7 +132,7 @@ def setup(self, **kwargs: Any) -> None:
         self._skill.skill_context.benchmark_tool.log_dir = Path(self.benchmark_dir.name)
         assert (  # nosec
             cast(BaseBehaviour, self.behaviour.current_behaviour).behaviour_id
-            == self.behaviour.initial_behaviour_cls.behaviour_id
+            == self.behaviour.initial_behaviour_cls.auto_behaviour_id()
```

Comment:
We will need to update all our tests, if we decide to remove the `behaviour_id` from them, ie see the benefit. We can obviously avoid that by leaving the `behaviour_id` where it already is.

Reply:
Yes if you want to upgrade you need to fix tests, o/w no change needed.